build.gradle

```diff
@@ -51,15 +51,16 @@ dependencies {
 
 	// hard dependency -> actually includes it in our jar
 	include "net.kyrptonaught:customportalapi:${project.portal_api_verion}"
-	include("maven.modrinth:Wd844r7Q:${project.structure_api_verion}")
+	include "maven.modrinth:Wd844r7Q:${project.structure_api_verion}"
+	include "curse.maven:DisableCustomWorldsAdvice-401978:${project.dcwa_file}"
 
 	// depend on source -> it needs to be installed alongside our mod or also be a hard dependency
 	modImplementation "net.kyrptonaught:customportalapi:${project.portal_api_verion}"
 	modImplementation "maven.modrinth:Wd844r7Q:${project.structure_api_verion}"
 
 	// debug dependency -> only used while running mod, not included or needed when released
 	modRuntimeOnly "mcp.mobius.waila:wthit:fabric-${project.wthit_version}"
-	modRuntimeOnly "curse.maven:DisableCustomWorldsAdvice-401978:${project.dcwa_file}"
+	modLocalRuntime "curse.maven:DisableCustomWorldsAdvice-401978:${project.dcwa_file}"
```

Comment:
Can you add a comment describing when we want to use this option (like the other ones). Also I'm not sure it is needed, it might just as well work with `modRuntimeOnly`. There's not a lot of documentation and haven't tested.

Reply:
Will this suffice: https://github.com/ScicraftLearn/Minelabs/pull/664/commits/0591d211901b72e1365a64444b39a36f0dda9753

It does work with `modRuntimeOnly`. The only reason I can find that we need this line is because of the way gradle makes/loom makes the `dev jar` it's basically a minimal jar with only our code in it. And since `DisableCustomWorldsAdvice` needs to be run alongside it, we have to call modRuntimeOnly. (no idea what local adds/removes)